### ISSUE TITLE
1.16.5 issue with bal after the bal top fix

### ISSUE THREAD

**Issue by genfar1980** [Gunpowder-MC/GunpowderCurrency#3]
The latest version for 1.16.5 has a issue with using /bal command and other things like signs.

Here is a crashlog that was generated.
[crash-2021-07-22_15.20.36-server.txt](https://github.com/Gunpowder-MC/GunpowderCurrency/files/6865245/crash-2021-07-22_15.20.36-server.txt)

**Comment by martmists-gh** [comment]
Does the issue persist in the 1.17 build?

**Comment by BushTrocity** [comment]
Yes and in single player. Balance works fine for me. If i try to buy, sale or do anything that interacts with the database will crash.

**Comment by genfar1980** [comment]
For me going to 1.17 is not a option, most of the mods I use have not updated to that version, also then you got 1.18 comming which will again mess up exsisting generation of worlds as depth and height of the worlds will get changed. Also most of the mods in my server I would want them. I personaly am not in any way at the moment good enough to just take the source code and make fixxes  I personalnly been trying to figure if there is a json way to just disable to use of balance top, as I went back to that version because at least outside of that one crash the rest of the mod is usable.

**Comment by dandin87** [comment]
Player Roles is the mod we use on the AllOfFabric servers to control what commands players can use. I there is also another mod that allows you to blacklist auto complete commands, but player roles will actually prevent their use.
  https://www.curseforge.com/minecraft/mc-mods/player-roles-fabric

**Comment by martmists-gh** [comment]
Removed the above comment because it doesn't actually solve the issue, it's just a temporary workaround.

**Comment by genfar1980** [comment]
Anyhow while this is being investigated or whatnot I need to really know  what the permission line is for balance top  putting  currency.balance.top  false   into luckperms does not prevent players from using balance top.  Really need to disable the top command.     Also luckperms does not seem to even notice commands ran so couldnt use verbose to get the permission listing.

**Comment by martmists-gh** [comment]
Don't use false, just don't add currency.balance.top at all. Permissions with patbox' API don't use key-value pairs, only permission nodes being present or not.

**Comment by genfar1980** [comment]
its not really a luckperms issue  its more a fabric issue    at least thats what I read. More so also player roles accualy worked where luckperms did not because player roles does its stuff differnt. But yea outside the point I will just use the player roles as a temp fix as now no one on my server can use the top command for balance   bal or money veriations of the command.
